src/modules/organizations.ts

```diff
@@ -15,16 +15,21 @@ import type { FieldData } from "./fields.js";
 export interface IOrganization extends Entity, TreeNode, Fields {
```

Comment:
`IOrganization` should not implement `TreeNode` or `Fields` as the underlying data models do not implement these behaviours.